auth/jwt/token_test.go

```diff
@@ -57,7 +57,7 @@ func TestParse(t *testing.T) {
 	require.Nil(t, err, fmt.Sprintf("issuing key expected to succeed: %s", err))
 
 	apiKey := key()
-	apiKey.Type = authn.APIKey
+	apiKey.Type = auth.APIKey
 	apiKey.ExpiresAt = time.Now().UTC().Add(-1 * time.Minute).Round(time.Second)
 	apiToken, err := tokenizer.Issue(apiKey)
```

Comment:
if you will follow naming, then token makes more sense than tokenizer (interface name).